front/lib/api/assistant/generation.ts

```diff
@@ -601,7 +601,7 @@ export async function* runGeneration(
     return;
   }
 
-  const { eventStream } = res.value;
+  const { eventStream, dustRunId } = res.value;
```

Comment:
This file disappears today no need to handle

Reply:
typings